src/com/company/Main.java

```diff
@@ -230,6 +237,45 @@ public static void main(String[] args) {
 
   }
 
+  private static void addPhoneNumbers (PhoneNumber phoneNumber){
+
+    long n=10;
+    int a[]=new int[10];
+    Scanner sc=new Scanner(System.in);
+    System.out.println("Enter a digits of phone number");
+    for(int i=0;i<n;i++)
+    {
+
+      a[i]=sc.nextInt();
+      if(a[i]<0 || a[i]>9)
+      {
+
+        System.out.print("wrong input");
+        return;
+      }
+
+      System.out.println("");
+
+    }
```

Comment:
Instead of that much work, 
I think we can get the phone number as whole
like entering 3459999999 at once